src/Authentication.Abstractions/Interfaces/IPSCmdletAction.cs

```diff
@@ -0,0 +1,23 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+
+namespace Microsoft.Azure.Commands.Common.Authentication.Abstractions
+{
+    public interface IPSCmdletAction
```

Comment:
The name `IPSCmdletAction` looks too general to me. Can we be more specific so it's more obvious about its function? For example `ITestCoverageUtility` or `ITestCoverageHelper`

Reply:
Moved IPSCmdletAction from Abstraction to Common and renamed to ITestCoverage.